xcube/core/store/stores/memory.py

```diff
@@ -76,7 +79,7 @@ def search_data(self, type_id: str = None, **search_params) -> Iterator[DataDesc
                 yield new_data_descriptor(data_id, data)
 
     def get_data_opener_ids(self, data_id: str = None, type_id: str = None) -> Tuple[str, ...]:
-        return '*:*:memory',
+        return _ACCESSOR_ID,
```

Comment:
Looking at the the docstring, this should not return a pattern but a list of ids. Same is valid for get_data_writer_ids below.

Reply:
This has been in there since the beginning, and we talked about it, see the predicate factory function. Just using a constant now.